examples/connext_dds/property_qos/java/build.gradle

```diff
@@ -0,0 +1,8 @@
+plugins {
+  id 'com.github.rticommunity.connext-dds-build-example'
+}
+
+ConfigureExample {
+  idlFile = 'numbers.idl'
+  codegenExtraArgs = ['-update', 'examplefiles']
```

Comment:
we are not consistent with the way we add new arguments, sometimes we use simple quote `'` and other times we use double quote `"`